sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/KotlinTreeVisitor.java

```diff
@@ -230,6 +243,35 @@ private Tree createParenthesizedExpression(TreeMetaData metaData, KtParenthesize
     Token rightParenthesis = toSlangToken(element.getLastChild());
     return new ParenthesizedExpressionTreeImpl(metaData, expression, leftParenthesis, rightParenthesis);
   }
+  private Tree createReturnTree(TreeMetaData metaData, KtReturnExpression element) {
+    Tree returnExpression = null;
```

Comment:
there is ambiguity what is `returnExpression` (as Kotlin uses it for entire `return foo`), rename it to `returnedExpression` or `returnBody`